### PR TITLE
Feat/listening to events

### PR DESCRIPTION
**Pull request by hojarasca**

### CODE REVIEW

File: cli/src/lib/block-explorer-client.ts [hunk 1]
```diff
@@ -148,6 +153,48 @@ export class BlockExplorerClient implements BlockExplorer {
     }
   }
 
+  async getLogs(
```
Comment (hojarasca) [inline cli/src/lib/block-explorer-client.ts:156]: I ended up not using this for now, but I'm sure that it's going to be needed for real production.

File: webapp/.env.example [hunk 2]
```diff
@@ -4,6 +4,7 @@ LOG_LEVEL=info
 NODE_ENV=development
 WALLET_CONNECT_PROJECT_ID=:placeholder:
 L1_RPC_URL=":placeholder:"
+L1_RPC_URL_FOR_UPGRADES="http://127.0.0.1:8545"
```
Comment (hojarasca) [inline webapp/.env.example:7]: anvil default url

File: webapp/app/.server/db/custom-types.ts [hunk 3]
```diff
@@ -15,6 +15,6 @@ export const bytea = customType<{
     return Buffer.from(newVal, "hex");
   },
   fromDriver(val) {
-    return val.toString("hex");
+    return `0x${val.toString("hex")}`;
```
Comment (hojarasca) [inline webapp/app/.server/db/custom-types.ts:18]: I found that we were recieving and returning in diffrent format, which was confusing

File: webapp/vite.config.ts [hunk 4]
```diff
@@ -13,12 +13,6 @@ export default defineConfig({
     },
   },
   plugins: [
-    nodePolyfills({
-      include: ["buffer", "events", "http"],
-      globals: {
-        process: false,
-      },
-    }),
```
Comment (hojarasca) [inline webapp/vite.config.ts:21]: removed polyfills for now.

**Review by aon — APPROVED** [review]

**Pull request merged**